src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/hudsontrigger/GerritTrigger.java

```diff
@@ -208,6 +210,20 @@ public GerritTrigger(List<GerritProject> gerritProjects) {
             this.notificationLevel = "";
         }
 
+        try {
+          DescriptorImpl descriptor = (DescriptorImpl)getDescriptor();
+          if (descriptor != null) {
+              ListBoxModel options = descriptor.doFillSilentLevelItems();
+              if (!options.isEmpty()) {
+                  this.silentLevel = SilentLevel.valueOf(options.get(0).value);
+              }
+          }
+          //CS IGNORE EmptyBlock FOR NEXT 1 LINES. REASON: Handled one row below
+        } catch (NullPointerException ignored) { /*Could happen during testing*/ }
+        if (this.silentLevel == null) {
+            this.silentLevel = SilentLevel.ALL;
```

Comment:
I'm not liking this default, the default should be that it behaves like it always has.